dpp4/api/routes.py

```
@@ -5,11 +5,16 @@
 """
 
 from typing import Dict, List, Optional, Any
-from flask import Flask, request, jsonify, Blueprint, current_app
+from flask import Flask, request, jsonify, Blueprint, current_app, send_file
 from http import HTTPStatus
+import os
+import tempfile
+import json
```

Ruff (0.8.2)

`json` imported but unused

Remove unused import: `json`

(F401)